code/modules/clothing/spacesuits/rig/suits/combat.dm

```diff
@@ -102,14 +102,6 @@
 	name = "blackshield hazard hood"
 	desc = "A hazard hood modded to help against CQC combat."
 	camera_networks = list(NETWORK_SECURITY)
-	armor_list = list(
```

Comment:
why are you nerfing bs hardsuit?

Reply:
there's 0 reason for the helmet to have extra armor on top of its already high armor.